From f6d965a9f81b3062ca2f9194bcfa52ee4eeec928 Mon Sep 17 00:00:00 2001
From: wangjiezhe <wangjiezhe@gmail.com>
Date: Sat, 23 Dec 2023 10:06:57 +0800
Subject: [PATCH 1/2] Revert "Exclude `extractor_wrapper` and
 `generator_wrapper` from the pip package, and remove the exclusion for the
 deleted `create_python_api_wrapper`."

This reverts commit bd50049c3905fd8c9ad058d2e09682b45b9b9b02.
---
 tensorflow_estimator/tools/pip_package/create_pip_helper.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/tensorflow_estimator/tools/pip_package/create_pip_helper.py b/tensorflow_estimator/tools/pip_package/create_pip_helper.py
index 4810594..e1b1f27 100644
--- a/tensorflow_estimator/tools/pip_package/create_pip_helper.py
+++ b/tensorflow_estimator/tools/pip_package/create_pip_helper.py
@@ -35,8 +35,7 @@ PIP_EXCLUDED_FILES = frozenset([
     'tensorflow_estimator/python/estimator/canned/v1/baseline_test_v1.py',
     'tensorflow_estimator/python/estimator/canned/v1/dnn_test_fc_v1_v1.py',
     'tensorflow_estimator/python/estimator/canned/v1/dnn_test_fc_v2_v1.py',
-    'tensorflow_estimator/python/estimator/api/extractor_wrapper.py',
-    'tensorflow_estimator/python/estimator/api/generator_wrapper.py',
+    'tensorflow_estimator/python/estimator/api/create_python_api_wrapper.py',
     'tensorflow_estimator/tools/pip_package/setup.py',
     'tensorflow_estimator/tools/pip_package/create_pip_helper.py',
 ])
-- 
2.41.0

